qemu/qemu.go

```diff
@@ -131,6 +134,9 @@ func (object Object) QemuParams(config *Config) []string {
 	var qemuParams []string
 
 	deviceParams = append(deviceParams, string(object.Driver))
+	if object.DisableModern {
```

Comment:
Maybe I am mistaken. I do not think we can add it blindly. We need to do this on a per device type.